reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -85,70 +187,38 @@ Only the following comparison operators are permitted:
 - `-gt` (greater-than)
 - `-lt` (less-than)
 
-Assignment statements, property references, and method calls are not
+Assignment statements, property references, and method calls aren't
 permitted.
 
-#### No language (NoLanguage)
-
-**NoLanguage** mode can only be used through the API. NoLanguage mode means no
-script text of any form is permitted. This precludes the use of the
-**AddScript()** method which sends fragments of PowerShell script to be parsed
-and executed. You can only use **AddCommand()** and **AddParameter()** which
-don't go through the parser.
-
-Beginning in PowerShell 7.2, the `New-Object` cmdlet is disabled in
-**NoLanguage** mode when system lockdown is configured.
-
-#### Constrained language (Constrained Language)
-
-The **ConstrainedLanguage** mode permits all cmdlets and all PowerShell
-language elements, but it limits permitted types.
+### ConstrainedLanguage mode
 
-**ConstrainedLanguage** mode is designed to support User Mode Code Integrity
-(UMCI) on Windows RT. It is the only supported language mode on Windows RT, but
-it is available on all supported systems.
+`ConstrainedLanguage` mode is designed to allow basic language elements such as
+loops, conditionals, string expansion, and access to object properties. The
+restrictions prevent operations that could be abused by a malicious actor.
 
-UMCI protects ARM devices by allowing only Microsoft-signed and
-Microsoft-certified apps to be installed on Windows RT-based devices.
-**ConstrainedLanguage** mode prevents users from using PowerShell to circumvent
-or violate UMCI.
+The `ConstrainedLanguage` mode permits all cmdlets and a subset of PowerShell
+language elements, but limits the object types that can be used.
 
-The features of **ConstrainedLanguage** mode are as follows:
-
-- All cmdlets in Windows modules, and other UMCI-approved cmdlets, are
-  fully functional and have complete access to system resources, except as
-  noted.
+The features of `ConstrainedLanguage` mode are as follows:
 
+- All cmdlets in Windows modules are fully functional and have complete access
+  to system resources, except as noted.
 - All elements of the PowerShell scripting language are permitted.
-
 - All modules included in Windows can be imported and all commands that the
   modules export run in the session.
```

Comment:
Nit: This seems like it could be combined with the first bullet.

Also, if language modes work at all on not-Windows, I assume it's only the commands that ship in PowerShell that are available?